assets/js/modules/tagmanager/setup.js

```diff
@@ -189,51 +233,61 @@ class TagmanagerSetup extends Component {
 				usageContext,
 			};
 
-			let errorCode = false;
-			let errorMsg = '';
 			const { accounts, containers } = await data.get( TYPE_MODULES, 'tagmanager', 'accounts-containers', queryArgs );
 
-			if ( ! selectedAccount && 0 === accounts.length ) {
-				errorCode = 'accountEmpty';
-				errorMsg = __(
-					'We didn’t find an associated Google Tag Manager account, would you like to set it up now? If you’ve just set up an account please re-fetch your account to sync it with Site Kit.',
-					'google-site-kit'
-				);
-			}
-
-			// Verify if user has access to the selected account.
-			if ( selectedAccount && ! accounts.find( ( account ) => account.accountId === selectedAccount ) ) { // Capitalization rule exception: `accountId` is a property of an API returned value.
-				data.invalidateCacheGroup( TYPE_MODULES, 'tagmanager', 'accounts-containers' );
```

Comment:
Just double-checking, why was this removed? I assume it's because we shouldn't cache this anyway?

Reply:
I believe this was added before we updated all data requests during setup to not be cached. Using cache is opt-in so this is redundant since all data requests do not set `nocache = false` as the fourth parameter.